apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```
@@ -851,7 +861,9 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
 
         elif grading_method is GradingMethodType.DAG:
             depends_graph, group_belonging = extract_dag(true_answer_list)
-
+            misordered_blocks = get_wrong_order_blocks(submission,  depends_graph, group_belonging, tags_with_feedback)    
```

Ruff (0.8.2)

Multiple spaces after comma

Replace with single space

(E241)